docs/changelog.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.0.1 / 2020-05-13
```

Comment:
does this warrant `1.1.0` since there are a couple `Added` items ?

Reply:
:+1: for `1.1.0`

Reply:
especially because of `* [Added] Bump openapi-generator to 4.3.1. See [#182](https://github.com/DataDog/apigentools/pull/182).`